### PR TITLE
Frozensuffix

### PR DESCRIPTION
**Pull request by WhiteNervosa**
Frozen suffix appears: 
VrItQXD

Hopefully checking off a box https://github.com/MemeLabs/chat-gui/issues/82

### CODE REVIEW

File: assets/chat/js/const.js [hunk 1]
```diff
@@ -59,7 +59,9 @@ const GENERIFY_OPTIONS = {
     // 'dank': 'generify-dank',
     // 'hyper': 'generify-hyper generify-rustle',
     'love': 'generify-love',
-    'spin': 'generify-spin'
+    'spin': 'generify-spin',
+    'frozen' : 'frozen' // Encases an emote in ice 
+                        // https://user-images.githubusercontent.com/50147539/60741909-e32af280-9f30-11e9-91ed-e500151bfb84.jpg
```
Comment (jbpratt78) [inline assets/chat/js/const.js]: We can remove the comments added.

Comment (jbpratt78) [inline assets/chat/js/const.js]: Lets remove comments added

Comment (WhiteNervosa) [reply assets/chat/js/const.js]: Done :+1:

**Comment by WhiteNervosa** [comment]
> Feature-Approve
> 
> * Did someone test with wide/irregular emotes?

Yes, Xymos did here (which I should have done, sooner) and the only one that appears to have slight clipping issues is AngelThump https://github.com/MemeLabs/chat-gui/pull/114

**Comment by xDashh** [comment]
> > Feature-Approve
> > 
> > * Did someone test with wide/irregular emotes?
> 
> Yes, Xymos did here (which I should have done, sooner) and the only one that appears to have slight clipping issues is AngelThump #114

That was only regarding `:wide`, right? I meant regarding `:frozen`, i.e. is the freeze-block on the left wing of e.g. `Angelthump` or properly in the middle? (or does it minimize the entire emote into the ice?)

**Comment by WhiteNervosa** [comment]
OH - pardon, I get these mixed up too easy.

It works well with ResidentSleeper but that is only 52px, as it also shrinks the emotes a little bit. I can see a couple outliers doing poorly with it.

**Comment by jbpratt78** [comment]
Merged, thank you for the feature!

**Comment by xDashh** [comment]
Please note https://github.com/MemeLabs/chat-gui/issues/121

**Pull request merged**